db.go

```diff
@@ -514,13 +672,13 @@ func (db *DB) compact() (err error) {
 	return nil
 }
 
-func (db *DB) getBlock(id ulid.ULID) (*Block, bool) {
-	for _, b := range db.blocks {
+func getBlock(loaded []*Block, id ulid.ULID) *Block {
```

Comment:
I preferred previous approach more, because it gives the guarantee that block cannot be nil. It's more explicit. But not a blocker.

Reply:
`func (db *DB) getBlock(id ulid.ULID) (*Block, bool) {`

Reply:
similar to `map` when you do `_, ok := someMap[...]`

Reply:
Yep, now looking at the code I agree, it would be better with the old approach and more familiar to the behavior of maps.

Reply:
:+1:

Reply:
updated